android-project/android.export.sym

```diff
@@ -0,0 +1,6 @@
+{
+  global:
+    SDL_main;
+    JNI_OnLoad;

```

Comment:
```suggestion
    # JNI_OnLoad;  # uncomment this line when linking to a static SDL3 library
```

We advise users to link against a shared SDL3 library.
Leaving this line present will result in link warnings/errors if no such symbol can be found.

Reply:
What if you add the following to `android-project/app/jni/src/CMakeLists.txt`?
```
if(ANDROID)
    target_link_options(main PRIVATE "-Wl,-u,JNI_OnLoad")
```
And remove all new version script related changes.

Reply:
That line might perhaps be added to the main `CMakeLists.txt` instead:
```
if(ANDROID)
    target_link_options(SDL3-static INTERFACE "-Wl,-u,JNI_OnLoad")
endif()
```

(Please test)